.github/workflows/linting.yml

```diff
@@ -0,0 +1,19 @@
+name: Linting Salt-Toaster
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```

Comment:
Is there a way to prevent hardcoding python version here?

Reply:
We can try but do we want to really leave this to chance? I mean afaik `pylint` produces different output for different Python versions, doesn't it?

Reply:
Okay yes it would work: https://github.com/actions/setup-python/blob/main/action.yml#L8

Reply:
Well, I just want to prevent us from having to fix this file again once Python 3.6 is not the default one anymore (EOL 23 Dec 2021).

Maybe we could try `3.x` here.

Reply:
Hm, this is tricky. We will continue to use Python 3.6 after its upstream EOL outside of the bundle, but the Ubuntu image won't use our maintained package. With `setup-python` GitHub uses the versions from https://github.com/actions/python-versions afaik, which is different from our exact Python binary anyway.

I'm not sure about the best approach. Using `3.x` could mess with the reproducibility (e.g. today with Python 3.10's release) but we might not want to keep an outdated version around. We could use 3.9, which is what we use in the bundle as a compromise. It'll take a while until it reaches EOL and still is fixed.

Reply:
Please remember that this is just for linting the code we maintain in the salt-toaster. So this should only matter for the `pylint` output, not for any runtime code we have.

Reply:
So what do we want to have now?

Reply:
For me this not really important, pick what you prefer. If we e.g. go with 3.x and see that we get too many failures due to Python 3.10, we can reconsider anyway.